.github/workflows/chart-lint-and-installation.yaml

```
@@ -1,12 +1,9 @@
 # validate any chart changes under charts directory
-name: Chart Lint
+name: Chart Lint and Installation
```

```suggestion
name: Chart
```
Avoid stutter like `Chart Lint and Installation / chart-lint-and-installation-test (pull_request) `.